source/Reference.js

```diff
@@ -2,4 +2,5 @@ module.exports.GAME_WIDTH = 230
 module.exports.GAME_HEIGHT = 130
 module.exports.JOIN_GAME_TEXT = "PRESS TO JOIN"
 module.exports.JUNK_FREQUENCY_RANGE = { lower: 5, upper: 10}
-module.exports.TIME_TO_DESPAWN = 5 * 1000
\ No newline at end of file
+module.exports.TIME_TO_DESPAWN = 5 * 1000
+module.exports.WEAPON_RANGE = { lower: 0, upper: 10}
```

Comment:
We shouldn't have to hardcode "lower" bound for weapons; it'll always be 0 to MAX_WEAPON_COUNT, won't it?

Even better, why can't we use `WeaponList.length` to query how many possible weapons we can randomize between? That way, we don't have to keep maintaining this number as we remove and add more weapons.